Source/ablastr/utils/SignalHandling.cpp

```diff
@@ -134,12 +135,23 @@ SignalHandling::InitSignalHandling()
                                                "Failed to install signal handler for a configured signal");
         }
     }
+
+    for (int signal_number = 0; signal_number < NUM_SIGNALS; ++signal_number) {
+        for (int signal_request = 0; signal_request < SIGNAL_REQUESTS_SIZE; ++signal_request) {
+            m_any_signal_action_active |= signal_conf_requests[signal_request][signal_number];
```

Comment:
You can just set this to true at line 128

Reply:
Hm, cannot follow, do you mean line 138?
Even then I am not sure.

Reply:
I was suggesting inside the `if (signal_active) {` block - i.e. exactly where we're already checking the array entries

Reply:
ah, of course. good idea :)